app/views/content/life-as-a-teacher/explore-subjects/english/_article.html.erb

```diff
@@ -0,0 +1,91 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach English</h2>
+    <div class="inset">
+      <p>As an English teacher, you'll be a cornerstone in pupils' lives. From reading and exploring literature to writing and delivering debates, you'll cover essential skills needed to think critically and thrive beyond the classroom.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <!-- Quote: Chris Farr -->
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching English opens the door for your pupils to pursue exciting careers in fields such as journalism, marketing and publishing. You may even inspire the next generation of teachers.</p>
+      <p>Your pupils will learn more from you than you might think. As their teacher, you'll guide them through the skills they need to show their understanding of written and spoken English. You'll teach them valuable life skills such as critical thinking, creativity and communication.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <!-- Quote: Holly, science teacher -->
+    <%= render 'content/shared/quotes/quote_english_laura' %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-english-programmes-of-study/national-curriculum-in-england-english-programmes-of-study">national curriculum for English</a>.</p>
```

Comment:
In the section below, key stage 3 the key stage is in brackets, but it's the other way round for key stage 4, with the ages in brackets. On other pages, the key stage is in brackets, so I'd maybe make this consistent too.